src/app/caesar-portal/caesar-home/caesar-home.component.ts

```diff
@@ -15,9 +15,20 @@ export class CaesarHomeComponent implements OnInit {
 	}
 
 	ngOnInit () {
-		
+		this.checkEndingSession();
     }
 
-    
+    checkEndingSession () {
+    	if (localStorage.getItem('loggedUser')) {
+    		this.http.get(environment.serviceApi.groupsUrl)
```

Comment:
it's ok but it'd be better to have separate API to refresh token instead of calling groups API